app/networkapi/settings.py

```diff
@@ -45,7 +46,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env('DEBUG')
 
-ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS").split(",")
```

Comment:
I can use `environ` on my windows machine, so that sounds like it might be a different problem